content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Quick Start with Nested Cluster"
+metaTitle: "Getting Started with nested clusters on Palette"
+metaDescription: "The methods of creating nested clusters for a speedy deployment on any CSP"
+icon: "clusters"
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import InfoBox from 'shared/components/InfoBox';
+import WarningBox from 'shared/components/WarningBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+# Getting Started with Nested Clusters
+
+With the Palette Nested Cluster option, you can deploy additional Kubernetes clusters within an existing cluster to provide teams with sandboxed environments where you are free to operate with admin level privileges, while simultaneously ensuring strong isolation, reducing operational overhead, and improving resource utilization.
+
+To access a Palette Sandbox, go to [SpectroCloud](https://spectrocloud.com) <!--confirm link--> and use your Github or Google SSO. You will be prompted to create a Palette account.
+You are allowed to deploy a managed Nested Cluster. See below under [Deploying a Nested Cluster](/clusters/nested-clusters/cluster-quickstart#deployinganestedcluster), follow the steps below under 
+
+
+# Prerequisites
+
+- You will need to have access to a Palette Account. Refer to the [Getting Started with Palette](/getting-started).
+
+
+- Ensure you have a functioning cluster already configure. Refer to the [New Clusters](/clusters/new-clusters) page for more information.
```

Comment:
1. Enable the nested cluster option needs to be enabled from the settings tab.

2. Cluster should be launched with this settings.